manifest/fixtures/v2s1-invalid-signatures.manifest.json

```diff
@@ -1,6 +1,6 @@
 {
    "schemaVersion": 1,
-   "name": "mitr/buxybox",
+   "name": "mitr/busybox",
```

Comment:
I could be wrong

Reply:
You’re right.

Reply:
Note to self: These kind of changes do affect some of the digest constants, and I have _not_ reviewed that so far.